.github/workflows/code.yml

```diff
@@ -104,14 +104,18 @@ jobs:
           BUILD_ARGS: ${{ matrix.test.build-args }}
 
       - name: Run bash e2e tests
-        run: utils/e2e-tests/bash/run-tests "$TEST_SUITE" "target/release/humanode-peer" "target/release/wbuild/humanode-runtime/humanode_runtime.wasm"
+        run: utils/e2e-tests/bash/run-tests "$TEST_SUITE"
         env:
           TEST_SUITE: ${{ matrix.test.suite }}
+          HUMANODE_PEER_PATH: "target/release/humanode-peer"
+          RUNTIME_WASM: "target/release/wbuild/humanode-runtime/humanode_runtime.wasm"
+          LIVE_NETWORK_URL: "wss://explorer-rpc-ws.it6.stages.humanode.io:443"
```

Comment:
We should probably run against mainnet

Reply:
https://github.com/humanode-network/humanode/pull/963/commits/bcdb75cd1e306072e39e5f9c13e898517cdef104